array/array.mbt

```diff
@@ -580,6 +580,43 @@ pub fn to_list[T](self : Array[T]) -> List[T] {
   }
 }
 
+/// Concat two arrays.
+/// 
+/// # Example
+///
+/// ```
+/// [1, 2, 3, 4, 5].concat([6, 7, 8, 9, 10]) //[1, 2, 3, 4, 5, 6, 7, 9, 10]
+/// ```
+pub fn concat[T](self : Array[T], last : Array[T]) -> Array[T] {
+  if self.length() == 0 {
+    last
```

Comment:
the semantics is questionable since here the returned result is shared with the input, which means when I change the result, the input is changed?

Reply:
`op_add`?

Reply:
Yes, this was my mistake. I would like to ask, is op-add reasonable and allowed for the design of the Array?